docs/Developer_Quickstart.md

```diff
@@ -141,56 +138,53 @@ this dApp, you do not need to change any Solidity files.
     truffle migrate --network arbitrum
     ```
 
-2. Deploy `contract.ao` to 3 Validators
+2. Create a rollup chain and initialize 3 validators with the given `contract.ao`. This command will initialize your rollup chain and create a `validator-states` folder with configuration information on how to run your chain.
```

Comment:
Is it right to say this "initialize[s] ... validators"?  That sounds like you're starting the validators running, when actually you're just creating config info that you'll use to launch the validators.